roles/prometheus/templates/server.yaml

```diff
@@ -0,0 +1,210 @@
+---
+# Source: prometheus/templates/server-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: prometheus
+    release: prometheus-{{ k8s_prometheus_image_tag }}
+    component: "server"
+  namespace: {{ k8s_prometheus_namespace }}
+  name: prometheus-server
+data:
+  alerts: |-
+{% include 'alerts/common.conf' %}
+
+  prometheus.yml: |-
+    # Rule files specifies a list of globs. Rules and alerts are read from
+    # all matching files.
+    rule_files:
+      - /etc/config/alerts
+
+    # A list of scrape configurations.
+    scrape_configs:
+{% include 'scrape_configs/prometheus.yml' %}
+
+{% include 'scrape_configs/kubernetes.yml' %}
+
+{% include 'scrape_configs/istio.yml' %}
+
+---
+# Source: prometheus/templates/server-serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: prometheus
+    release: prometheus-{{ k8s_prometheus_image_tag }}
+    component: "server"
+  namespace: {{ k8s_prometheus_namespace }}
+  name: prometheus-server
+
+---
+# Source: prometheus/templates/server-clusterrole.yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    app: prometheus
+    release: prometheus-{{ k8s_prometheus_image_tag }}
+    component: "server"
+  name: prometheus-server
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+      - nodes/proxy
+      - services
+      - endpoints
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+  - nonResourceURLs:
+      - "/metrics"
+    verbs:
+      - get
+
+---
+# Source: prometheus/templates/server-clusterrolebinding.yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: prometheus
+    release: prometheus-{{ k8s_prometheus_image_tag }}
+    component: "server"
+  namespace: {{ k8s_prometheus_namespace }}
```

Comment:
ClusterRoleBinding can not have any namespace

Reply:
fixed